wpilibj/src/main/java/edu/wpi/first/wpilibj/PWM.java

```diff
@@ -261,5 +261,7 @@ public void initSendable(SendableBuilder builder) {
     builder.setActuator(true);
     builder.setSafeState(this::setDisabled);
     builder.addDoubleProperty("Value", this::getRaw, value -> setRaw((int) value));
+    builder.addDoubleProperty("Speed", this::getSpeed, value -> setSpeed(value));
```

Comment:
Replace with method reference

Reply:
I'm not sure it makes sense to change the style in this PR, as I believe it's used throughout the library for SendableBuilder

Reply:
never mind